src/i18n.jl

```diff
@@ -0,0 +1,58 @@
+still_nothing_str(lang::Lang) where {Lang <: English} = "Here we go!"
+still_nothing_text_str(lang::Lang) where {Lang <: English} = md"Replace `nothing` with your answer."
+wrong_type_str(lang::Lang) where {Lang <: English} = "Type Error"
+wrong_type_text_str(lang::Lang) where {Lang <: English} = md"Check the type of your response."
+wrong_type_text_str(lang::Lang, var, type) where {Lang <: English} = md"The type of $var should be $type"
+func_not_defined_str(lang::Lang) where {Lang <: English} = "Oopsie!"
+func_not_defined_text_str(lang::Lang, func_name) where {Lang <: English} = md"Make sure that you define a function called **$(Markdown.Code(string(func_name)))**"
+var_not_defined_str(lang::Lang) where {Lang <: English} = "Oopsie!"
+var_not_defined_text_str(lang::Lang, variable_name) where {Lang <: English} = md"Make sure that you define a variable called **$(Markdown.Code(string(variable_name)))**"
+keep_working_str(lang::Lang) where {Lang <: English} = "Keep working on it!"
+keep_working_text_str(lang::Lang) where {Lang <: English} = md"The answer is not quite right."
+keep_working_update_str(lang::Lang) where {Lang <: English} = md"Make sure to update the cell setting $var."
+yays(lang::Lang) where {Lang <: English} = [md"Great!", md"Yay ❤", md"Great! 🎉", md"Well done!", md"Keep it up!", md"Good job!", md"Awesome!", md"You got the right answer!", md"Let's move on to the next part."];
+correct_str(lang::Lang) where {Lang <: English} = "Got it!"
+
+check_type_isa_missing_text_str(sym, lang::Lang) where {Lang <: English} = md"The variable \$sym is still set to missing."
+check_type_isa_wrong_type_text_str(sym, lang::Lang) where {Lang <: English} = "The type of \$sym is not correct.  It should be <: "
+check_type_isa_wrong_type_one_of_text_str(lang::Lang) where {Lang <: English} = "one of"
+check_type_isa_wrong_type_or_text_str(lang::Lang) where {Lang <: English} = "or"
+check_type_isa_not_missing_text_str(sym, lang::Lang) where {Lang <: English} = md"\$sym has the correct type."
+check_type_isa_type_error_str(sym, lang::Lang) where {Lang <: English} = "Type Error"
+
+check_type_eq_missing_text_str(sym, lang::Lang) where {Lang <: English} = md"The variable \$sym is still set to missing."
+check_type_eq_wrong_type_single_text_str(sym, type, lang::Lang) where {Lang <: English} = "The type of \$sym should be \$type."
+check_type_eq_wrong_type_multi_text_str(sym, lang::Lang) where {Lang <: English} = "The type of \$sym should be one of "
+check_type_eq_type_error_str(lang::Lang) where {Lang <: English} = "Type Error"
+check_type_eq_correct_str(sym, lang::Lang) where {Lang <: English} = md"\$sym has the correct type."
+
+# other.jl
+full_width_mode_str(lang::Lang) where {Lang <: English} = "Full Width Mode"
+
+# present.jl
+present_str(lang::Lang) where {Lang <: English} = "present"
+present_mode_str(lang::Lang) where {Lang <: English} = "Present Mode"
+
+default_language = Ref{AbstractLanguage}(EnglishUS())
```

Comment:
I'm not a frequent user of references in Julia.  Would you mind explaining why you used one here?

Reply:
This was mainly based on [this comment](https://github.com/tknopp/RedPitayaDAQServer/commit/cfa713ec2c1248c856603a0d91eb1b353ad885e0#r74307377). When looking deeper into this, I could not find details on this. I am also not sure how much penalty we get by having this variable type Any. With Julia 1.8, we could also directly annotate the type to avoid the instability but this would break compatability with older Julia versions.